### DISCUSSION TITLE
Needing to re-signin github every time webpage is refreshed (nginx reverse proxy and https)

### DISCUSSION THREAD

**Discussion by simonfalke-01** [coder/code-server#5477] [Q&A] [answered]
I have code-server running on my docker host, and the nginx reverse proxy in a docker container. I can access code-server with a domain name with https, but every time I refresh the page, I need to sign in to GitHub again. Is there anyway I can save the sign in info so I do not need to do it every time? Thanks!

**Comment by jsjoeio** [comment, marked as answer]
That sounds annoying! What version of code-server are you running? Any basic tips to repro?

**Reply by simonfalke-01** [reply]
code-server 4.5.1 97d170331b51ee75c8e4bbdab23c755e3eeaaa8c with Code 1.68
^^ Got the above from `code-server —help`

I have an nginx docker container running (exposed port 80), while code-server is running on host (port 8080). The nginx proxy listens on port 80 for connections from example.mydomain.com, then proxies the traffic to 192.168.2.50:8080. I am using LetsEncrypt certs, and I can access code-server from an external network by accessing example.mydomain.com. Everything is working fine, just the login issue.
If you need the nginx .conf file I can upload it here. Thanks!

**Reply by jsjoeio** [reply]
Hmm...I do get this notification in the corner on every refresh. I believe I have the `GITHUB_TOKEN` set in my `~/.zshrc`.

<img width="1712" alt="image" src="https://user-images.githubusercontent.com/3806031/185664949-dbe065e0-76ed-4711-b3a3-1b60e97d3044.png">

 🤔 Do you know if it's a specific extension asking for you to login? I'd try these solutions:
- set `GITHUB_TOKEN` in `.zshrc`
- uninstall extensions that could be causing this (to isolate problem)
- if possible, try removing nginx from equation (as in testing code-server by itself if possible) to see if causing issue

**Reply by simonfalke-01** [reply]
Setting `GITHUB_TOKEN` in `~/.zshrc` worked for me! Thanks!
